clients/da-vinci-client/classHash-1.txt

```diff
@@ -0,0 +1 @@
+-382233033
```

Comment:
Is this needed for testing?  Could we exclude it?

Reply:
No, this was committed by mistake. Thank you for catching this.